src/sentry/static/sentry/app/actionCreators/indicator.tsx

```diff
@@ -50,8 +50,7 @@ export function addMessage(
 }
 
 function addMessageWithType(type: IndicatorType) {
-  return (msg: React.ReactNode, duration?: number, options?: Omit<Options, 'duration'>) =>
-    addMessage(msg, type, {...options, duration});
+  return (msg: React.ReactNode, options: Options) => addMessage(msg, type,options)
```

Comment:
the options were not working because we were passing an object instead of 2 parameters...now it's fixed :)